src/lib/shared/components/ui/Jumbotron/Jumbotron.tsx

```diff
@@ -0,0 +1,89 @@
+import { Box } from "@mui/material";
+import { SxProps, Theme } from '@mui/material/styles';
+import { Button, Paragraph } from '@/lib/shared/components/ui';
+
+import { styled } from '@mui/material/styles';
+import { H1 } from '@/lib/shared/components/ui';
```

Comment:
Because we just exported Jumbotron from this file, were getting circular dependency issues now (that's why our tests and build is failing). We need to use relative imports here, since were in the same directory.